plugins/node-cert/commands/smoke-test.md

```
@@ -0,0 +1,74 @@
+# node-cert smoke-test
+
+Test Node.js HTTPS connectivity with current configuration.
+
+## Usage
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)